src/backend/src/router/triage_agent_router.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+import os
+import json
+import logging
+import pii_redacter
+from typing import Callable
+from azure.ai.agents import AgentsClient
+from azure.ai.agents.models import ListSortOrder
+from router.clu_router import parse_response as parse_clu_response
+from router.cqa_router import parse_response as parse_cqa_response
+from utils import get_azure_credential
+
+_logger = logging.getLogger(__name__)
+
+PII_ENABLED = os.environ.get("PII_ENABLED", "false").lower() == "true"
+
+def create_triage_agent_router() -> Callable[[str, str, str], dict]:
+    """
+    Create triage agent router.
+    """
+    project_endpoint = os.environ.get("AGENTS_PROJECT_ENDPOINT")
+    credential = get_azure_credential()
+    agents_client = AgentsClient(
+        endpoint=project_endpoint,
+        credential=credential,
+        api_version="2025-05-15-preview"
+    )
+    agent_id = os.environ.get("TRIAGE_AGENT_ID")
+    agent = agents_client.get_agent(agent_id=agent_id)
+
+    def triage_agent_router(
+        utterance: str,
+        language: str,
+        id: str
+    ) -> dict:
+        """
+        Triage agent router function.
+        """
+
+        # Create thread for communication
+        thread = agents_client.threads.create()
+        print(f"Created thread, ID: {thread.id}")
```

Comment:
nit - since we're importing the logger in this file - can we use `logger.info` everywhere instead of print statements?

Similar in other files